checks/evaluation/cii_best_practices.go

```diff
@@ -15,48 +15,69 @@
 package evaluation
 
 import (
-	"fmt"
-
 	"github.com/ossf/scorecard/v4/checker"
-	"github.com/ossf/scorecard/v4/clients"
 	sce "github.com/ossf/scorecard/v4/errors"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/hasOpenSSFBadge"
 )
 
-// Note: exported for unit tests.
 const (
 	silverScore = 7
 	// Note: if this value is changed, please update the action's threshold score
 	// https://github.com/ossf/scorecard-action/blob/main/policies/template.yml#L61.
 	passingScore    = 5
 	inProgressScore = 2
+	minScore        = 0
```

Comment:
not needed?

Reply:
Removed in https://github.com/ossf/scorecard/pull/3520/commits/5f1dde3560524b33a8639952a30bbd103c4ec5e0